config/initializers/proofer.rb

```diff
@@ -0,0 +1,8 @@
+# gems require UPPER case ENV variables so translate
+Idv::Vendor.new.available.each do |vendor|
+  ENV.keys.each do |env_var_name|
```

Comment:
what this does is pretty straightforward, but a test might be good just as a form of documentation. I can imagine someone trying to add a lowercase env var and not knowing why it is always getting uppercased